backend/src/document.ts

```diff
@@ -28,6 +28,7 @@ async function getDocuments(filePath: string) {
 		'.csv': (path: string) => new CSVLoader(path),
 	});
 	const docs = await loader.load();
+	console.debug(`${docs.length} documents found`);
```

Comment:
This is the line that exposed we weren't stubbing the loader to return anything. So I simply added that in my test changes.